tests/resources/metadata/sample_metadata1.json

```diff
@@ -3,31 +3,26 @@
         "name": "UK Weather information 2011",
         "type": "dataset",
         "description": "Weather information of UK including temperature and humidity",
-        "size": "3.1gb",
         "dateCreated": "2012-10-10T17:00:000Z",
         "author": "Met Office",
         "license": "CC-BY",
         "copyrightHolder": "Met Office",
-        "encoding": "UTF-8",
-        "compression": "",
-        "contentType": "text/csv",
         "workExample": "Text PDF",
-        "language": "en",
         "categories": ["white-papers"],
-        "tags": "weather, uk, 2011, temperature, humidity",
+        "tags": ["weather, uk, 2011, temperature, humidity"],
```

Comment:
Since this changed to being an array does each token want to be a separate string?

Reply:
oh yes, i will change..